mk/e2e.new.mk

```diff
@@ -45,6 +45,19 @@ test/e2e/test:
 	KUMACTLBIN=${BUILD_ARTIFACTS_DIR}/kumactl/kumactl \
 		$(GO_TEST) -v -timeout=45m ./test/e2e/...
 
+# test/e2e/debug is used for quicker feedback of E2E tests (ex. debugging flaky tests)
```

Comment:
Should we make this one the default `test/e2e` and just call the other one `test/e2e/all` and invoke it in the CI?

Reply:
I don't know... I think this should be symmetrical to `make test` which executes all the tests.
@lobkovilya what is your opinion here?

Reply:
I'd prefer to have an explicit `debug` word in the target to be honest, so I like how it is implemented in the PR